docs/pages/architecture/proxy-peering.mdx

```diff
@@ -0,0 +1,57 @@
+---
+title: Proxy Peering
+description: How Teleport implements more efficient networking with proxy peering.
+---
+
+<Details
+  title="Version warning"
+  opened={true}
+  scope={["oss", "enterprise"]}
+  scopeOnly={true}
+  min="10.0"
+>
+  Proxy peering is available starting from Teleport `10.0`.
+</Details>
+
+## Overview
+
+Proxy peering is a type of tunnel strategy configurable in Teleport. A tunnel
+strategy defines the behavior for agent reverse tunnel connections as well as
+how client are connected to those reverse tunnels. A Teleport Cluster's tunnel
+strategy is configurable in the Teleport Auth service.
+
+Proxy peering enables an agent to be reachable without it connecting to every
```

Comment:
- This seems like the main point of Proxy Peering—we should put this paragraph first.

- What is an "agent" in this context? I'm guessing its an instance of the Database Service, SSH Service, etc., but will a reader know that? (This applies to the term "agents" throughout this page.)

Reply:
Should we define agent in this page or swap it for `Teleport resource service` and provide the list of resources this refers to?